301-drupal8-vmss-glusterfs-mysql/azuredeploy.parameters.json

```diff
@@ -68,10 +68,10 @@
       "value": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/gluster-file-system/"
     },
     "templateLocation": {
-      "value": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/301-drupal8-vmss-glusterfs-mysql"
+      "value": "https://raw.githubusercontent.com/maniSbindra/azure-quickstart-templates/master/301-drupal8-vmss-glusterfs-mysql"
```

Comment:
Along with the azuredeploy,json, the path of install_drupal.sh is also relative to this path. When I replace "maniSbindra" with "Azure" I believe the travis ci is picking the old version of install_drupal.sh and hence the build is timing out after 2 hours. Once this change is merged will submit new pull request modifying the "maniSbindra" to "Azure".